lib/spectacles/schema_statements/postgresql_adapter.rb

```diff
@@ -68,7 +68,7 @@ def materialized_view_build_query(view, name = nil)
         definition = row["definition"].strip.sub(/;$/, "")
 
         options = {}
-        options[:data] = false if ispopulated == 'f'
+        options[:data] = false if ispopulated == 'f' || ispopulated == false
```

Comment:
Fixes a bug in which `:data => false` was not appearing in the schema using certain adapters.

Reply:
👍